domain/src/main/java/com/bottlerocketstudios/brarchitecture/domain/models/Links.kt

```diff
@@ -0,0 +1,6 @@
+package com.bottlerocketstudios.brarchitecture.domain.models
+
+data class Links(
+    val avatar: Link?
```

Comment:
@VanyaHuamanBottleRocket nit: Add trailing comma for less noisy future diffs :)

```suggestion
    val avatar: Link?,
```

Reply:
Updating